CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [next-version]
+
+### Changed
+
+- Internal: Upgrade to latest version of Preact.
+- Internal: Replace `react-modal-onfido` with latest version of `react-modal`.
```

Comment:
I think it will be clearer to integrators if we specify the version of Preact & `react-modal` even if just specifying the major version. Would also make it easier for us to find which SDK version we upgraded to Preact 10 if we ever need to look back for this info in the future.